action.yml

```diff
@@ -25,7 +25,7 @@ runs:
         COMMENT_BODY="${COMMENT_BODY//$'\n'/'%0A'}"
         COMMENT_BODY="${COMMENT_BODY//$'\r'/'%0D'}"
         echo "::set-output name=comment_body::$COMMENT_BODY"
```

Comment:
This should also be updated, I can open a followup PR.

https://github.blog/changelog/2022-10-11-github-actions-deprecating-save-state-and-set-output-commands/